### PR TITLE
aggregator proxy contract

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: evm/contracts/examples/ConversionRate.sol [hunk 1]
```diff
@@ -16,7 +16,7 @@ contract ConversionRate is ChainlinkClient, Ownable {
     uint256[] responses;
   }
 
-  uint256 public currentRate;
+  uint256 public currentAnswer;
```
Comment (se3000) [inline evm/contracts/examples/ConversionRate.sol]: Notably, this contract [can't specify that it implements](https://github.com/ethereum/solidity/issues/3514) the CurrentAnswerInterface.

File: evm/test/AggregatorProxy_test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        1,
+        [oc1.address],
+        [jobId1]
+      )
+
+      assert.equal(aggregator.address, await proxy.aggregator.call())
+    })
+
+    context('when called by the owner', () => {
+      it('pulls the rate from the new aggregator', async () => {
```
Comment (thodges-gh) [inline evm/test/AggregatorProxy_test.js]:
```suggestion
      it('sets the address of the new aggregator', async () => {
```

File: evm/test/AggregatorProxy_test.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    proxy = await h.deploy(SOURCE_PATH, aggregator.address)
+  })
+
+  it('has a limited public interface', () => {
+    h.checkPublicABI(artifacts.require(SOURCE_PATH), [
+      'aggregator',
+      'currentAnswer',
+      'setAggregator',
+      // Ownable
```
Comment (coventry) [inline evm/test/AggregatorProxy_test.js]: This should be removed or uncommented. It's present in `ConversionRate_test.js`, too.

Comment (se3000) [reply evm/test/AggregatorProxy_test.js]: It's not actually a method, just delimiting the methods that are inherited from Ownable.sol vs defined in the contract.

Comment (coventry) [reply evm/test/AggregatorProxy_test.js]: Oh, right. Maybe change the comment to `// Ownable methods:`, then.

**Pull request merged**